### PR TITLE
Initialize feed flow

### PR DESCRIPTION
**Pull request by RodrigoAD**

### CODE REVIEW

File: gauntlet/packages/gauntlet-solana-contracts/networks/.env.local [hunk 1]
```diff
@@ -1 +1,9 @@
-NODE_URL=http://127.0.0.1:8899
\ No newline at end of file
+NODE_URL=http://127.0.0.1:8899
+
+PROGRAM_ID_ACCESS_CONTROLLER=DzzjdPWNfwHZmzPVxnmqkkMJraYQQRCpgFZajqkqmU6G
+PROGRAM_ID_OCR2=HW3ipKzeeduJq6f1NqRCw4doknMeWkfrM4WxobtG3o5v
+PROGRAM_ID_DEVIATION_FLAGGING_VALIDATOR=EnUkr8ruCbCe1EidYghGMYanjz6ifXAxphWZmLMri9Zg
```
Comment (krebernisak) [inline gauntlet/packages/gauntlet-solana-contracts/networks/.env.local:5]: Which program IDs are these?
Why not use the ones shared for the `localnet` setup at `./contracts/artifacts/localnet/*-keypair.json`?

Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-solana-contracts/networks/.env.local]: Those were randomly generated, just removed them

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/write.ts [hunk 2]
```diff
@@ -57,10 +57,22 @@ export default class WriteOffchainConfig extends SolanaCommand {
...
-    const operatorConfigPublicKeys = aggregatorOperators.map((o) => rdd.operators[o].ocrConfigPublicKey[0])
+
+    const aggregatorOperators: any[] = aggregator.oracles
+      .map((o) => rdd.operators[o.operator])
+      .sort((a, b) => {
+        if (a.ocr2OnchainPublicKey[0] > b.ocr2OnchainPublicKey[0]) return 1
+        if (a.ocr2OnchainPublicKey[0] < b.ocr2OnchainPublicKey[0]) return -1
+        return 0
+      })
+    console.log(aggregatorOperators)
```
Comment (krebernisak) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/write.ts:68]: Debugging artifact?

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts [hunk 3]
```diff
@@ -3,45 +3,101 @@ import { TransactionResponse, waitExecute } from '@chainlink/gauntlet-solana'
...
     ]
   }
+
+  showRddInstructions = async () => {
+    logger.info(
+      `
+        Change the RDD ID with the new state address: 
+          - State Address: ${this.getReportStepDataById(FlowCommand.ID.contract(this.stepIds.OCR_2))}
+        Include the transmissions account into the contract information:
+          - Transmissions Account: ${this.getReportStepDataById(
```
Comment (krebernisak) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts:95]: These are the same thing, right? Let's label them the same, either "Address" or 'Account"

**Review by krebernisak — APPROVED** [review]

**Pull request merged**